### ISSUE TITLE
[Enhancement] Make it possible to use your own titlekeys.json

### ISSUE THREAD

**Issue by Uupo03** [dojafoja/FunKii-UI#13]
I want to provide my own titlekeys.json to FunKii-UI, but it keeps replacing it from the link i give it.

**Comment by Uupo03** [comment]
Found a workaround:
1. Delete the config file
2. Place your own titlekeys.json into the FunKii-UI folder
3. After launching go directly to the download tab without inputting a url

**Comment by Uupo03** [comment]
Reason i want this is that no titlekey sites work for me but i managed to crab a titlekeys.json with wayback machine.

**Comment by Uupo03** [comment]
Yeah, strange. It generates tickets for games with an online ticket avaliable (if there is no url selected), I should be able to use the program like this. :)

**Comment by dojafoja** [comment]
Without a key site url, you should only be able to get a title.tik if it's an update or no online ticket is available. Getting it to always generate a fake ticket should be as easy as modifying one s line of code in the source.

**Comment by Uupo03** [comment]
Alright, thanks. You can close this issue if you want now.

**Comment by Uupo03** [comment]
One final thing, I got around the issue of no tickets downloaded for games by editing the titlekeys.json to have the "ticket" value always set to 0, now no titles show up when choosing the "only show items with legit tickets" option.

**Comment by dojafoja** [comment]
I don't know how you changed the ticket value of all titles to 0 in titlekeys.json. I assume you wrote a script that automates the process, I hope lol. I also wrote a tiny Python script that does this for you. If that would be helpful for you in the future let me know and I will give you a copy.
